lib/srv/authhandlers.go

```diff
@@ -60,6 +60,8 @@ var (
 	)
 
 	prometheusCollectors = []prometheus.Collector{failedLoginCount, certificateMismatchCount}
+
+	errRoleFileCopyingNotPermitted = trace.AccessDenied("file copying via SCP or SFTP not permitted")
```

Comment:
seems odd to group this with a bunch of prometheus metrics

Reply:
True, I'll move it out of that block